trunk/NDHMS/nudging/module_stream_nudging.F

```diff
@@ -1136,14 +1178,20 @@ subroutine setup_stream_nudging(hydroDT)
       tt = whObsMiss(ii)
       ! Treat obs_static_to_strutct
       call mpp_land_bcast_int1d_root(obsTimeStr(tt)%obsStr(:)%obsStaticInd, iiImage)
-      call mpp_land_bcast_char1d_root(obsStaticStr(:)%usgsId, iiImage)
-      call mpp_land_bcast_int1d_root(obsStaticStr(:)%obsCellInd, iiImage)
-      call mpp_land_bcast_real_1d_root(obsStaticStr(:)%R, iiImage)
-      call mpp_land_bcast_real_1d_root(obsStaticStr(:)%G, iiImage)
-      call mpp_land_bcast_real_1d_root(obsStaticStr(:)%tau, iiImage)
-
-      nStatic = sum( (/ (1, cc=1,size(obsStaticStr(:)%obsCellInd)) /), &
-                     mask=obsStaticStr(:)%obsCellInd .ne. 0            )
+      call mpp_land_bcast_char1d_root(obsStaticStr_SoA%usgsId(:), iiImage)
```

Comment:
This line seems like it should be commented out right? Since it is repeated at line 1187

Reply:
...or just removed, since it's clearly redundant

Reply:
Ok, removed that line and removed a few other commented out code blocks